config.py

```diff
@@ -17,4 +18,5 @@ class DevelopmentConfig(Config):
 
 class TestingConfig(DevelopmentConfig):
     DEBUG = False
-    Testing = True
```

Comment:
`TESTING` should be True. This stops any exceptions being handled by the flask app object and allows them to propagate to the unittest module.